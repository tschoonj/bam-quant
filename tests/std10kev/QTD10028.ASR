File: C:\xrfdaten\2014KW30\chris\std10kev\std10028.spe
Start: 21.07.2014 16:04:45
Stop: 21.07.2014 16:06:25
BESSY: 298.9738024 298.5305876
IONI: 1.437403E-10 1.430745E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 69205
Run: 100000
Dead: 31
ICR: 111080
OCR: 0
Accumulated BESSY: 59688.678491
Accumulated Ioni: 0.000000028681217
N Accumulation: 200
Normfactor:  1.0076158e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ni
exp_x 92.0204010009766 92.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  28   1      7.472   6697597.      2589.     157.32
