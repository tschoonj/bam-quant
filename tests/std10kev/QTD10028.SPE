$DATA:
0 2000
16590
10822
4805
1432
374
65
11
8
6
3
5
3
1
0
1
14
124
1486
2004
1826
1772
1773
1677
1579
1539
1495
1424
1433
1391
1301
1336
1314
1288
1213
1190
1179
1154
1147
1063
1106
1052
1021
1056
1021
961
955
1020
969
1020
906
947
940
885
899
935
871
890
910
879
905
887
901
910
921
852
899
807
819
872
814
801
800
859
795
824
809
813
826
794
746
715
782
756
805
746
755
762
780
733
744
771
734
765
732
751
738
728
792
827
835
825
812
853
869
813
839
817
769
748
761
698
742
704
749
720
697
701
685
689
666
733
707
722
713
706
728
741
730
797
801
798
753
744
740
679
711
695
733
668
683
721
731
791
892
997
1128
1167
1120
1059
890
796
729
746
635
657
698
694
687
721
687
664
663
641
666
601
620
677
643
682
637
630
669
606
662
655
610
673
631
654
594
692
680
655
633
670
669
644
633
654
688
627
671
664
686
656
687
684
631
637
662
654
623
657
613
642
689
632
655
688
683
667
645
706
658
636
673
646
674
742
675
711
692
698
678
640
643
690
702
674
666
671
698
658
680
682
683
715
669
749
694
736
711
689
735
812
741
716
758
742
713
753
776
762
768
805
742
798
754
798
848
789
737
812
862
805
816
844
923
926
901
867
904
887
881
937
970
1096
1162
1359
1628
2005
2441
2765
2944
3012
2713
2308
1898
1594
1358
1209
1042
1088
1061
981
974
1102
1048
1126
1067
1078
1096
1122
1129
1155
1119
1170
1179
1235
1304
1310
1323
1400
1461
1551
1660
1674
1769
1765
1731
1803
1794
1673
1658
1593
1588
1542
1478
1400
1393
1425
1522
1422
1461
1614
1590
1602
1711
1722
1771
1880
1952
1943
2006
2227
2345
2361
2471
2630
2839
2991
3244
3511
3780
4102
4444
5003
5674
7027
9784
15583
28051
51693
95058
165489
269266
402799
551893
688307
785303
811867
763745
657099
513876
366066
237753
141565
77457
39352
18966
9274
4907
3100
2263
1904
1847
1708
1713
1748
1809
1878
1972
2075
2186
2367
2528
2896
3384
4094
5152
7253
10823
16976
26532
40033
57242
76368
93359
105834
110348
105746
93294
75359
56316
38244
24215
14218
7600
3948
1827
937
496
269
218
159
122
124
112
113
96
74
65
72
77
75
74
73
74
47
66
59
70
51
62
59
51
46
47
39
49
33
34
40
34
36
31
31
29
30
36
28
30
28
40
24
36
23
38
45
36
29
49
35
32
40
37
47
45
39
46
44
62
68
68
91
116
138
158
188
221
240
282
307
355
353
317
347
300
286
230
181
143
96
76
57
56
35
27
19
22
21
22
20
19
27
33
23
21
22
21
13
13
16
17
19
19
17
19
17
12
20
12
10
21
18
13
14
22
12
11
22
16
15
16
20
17
14
10
18
19
17
15
9
11
16
17
16
16
18
12
9
18
13
20
12
9
15
13
13
8
8
8
9
10
16
17
10
26
12
12
8
7
10
13
13
25
16
20
7
15
14
16
14
18
10
12
8
9
15
19
15
15
9
13
11
17
23
12
16
14
19
11
17
22
13
10
15
12
8
8
11
7
11
16
16
14
16
22
21
14
17
12
15
17
11
13
14
10
14
20
19
14
19
24
11
20
29
30
34
26
34
40
45
38
33
32
28
29
39
28
24
27
21
18
26
21
22
21
16
20
17
20
31
13
27
15
19
19
17
14
16
18
23
18
25
30
28
28
23
32
36
29
27
23
25
33
26
26
24
33
34
25
25
30
28
41
28
36
28
22
32
36
44
44
47
38
55
51
58
47
46
71
70
86
92
128
176
246
348
534
779
1148
1541
2228
2797
3514
4184
4694
5127
5484
5190
4829
4272
3717
2977
2305
1644
1143
792
489
307
220
106
73
66
32
42
41
42
32
53
45
46
57
69
79
122
132
205
264
372
510
662
829
1001
1242
1298
1448
1504
1494
1420
1252
1058
906
690
465
384
248
157
109
58
35
25
11
7
7
8
10
5
7
5
9
10
7
9
8
11
17
15
29
42
54
77
70
82
96
100
102
105
103
97
73
66
43
33
27
20
16
7
12
3
2
2
0
1
3
0
1
0
0
0
0
0
0
1
0
2
2
0
2
0
1
2
3
0
4
1
2
6
6
9
3
9
5
7
1
2
5
3
2
2
3
0
2
0
0
2
0
1
1
0
1
0
1
0
0
1
0
0
0
0
0
0
2
2
1
1
1
1
3
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
1
0
1
0
1
0
0
0
1
0
0
0
0
0
1
1
3
1
1
1
2
2
2
3
7
7
8
16
18
19
26
16
23
18
12
22
23
19
21
11
11
11
9
1
4
2
1
2
1
0
2
0
2
0
0
0
1
0
0
1
4
0
4
3
2
2
3
8
4
6
5
11
11
12
7
11
7
6
6
8
5
2
1
0
2
0
1
0
0
0
0
0
2
0
0
0
0
0
1
0
0
1
0
3
0
0
2
1
1
3
2
0
2
2
0
2
3
2
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10028.spe
Start: 21.07.2014 16:04:45
Stop: 21.07.2014 16:06:25
BESSY: 298.9738024 298.5305876
IONI: 1.437403E-10 1.430745E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 69205
Run: 100000
Dead: 31
ICR: 111080
OCR: 0
Accumulated BESSY: 59688.678491
Accumulated Ioni: 0.000000028681217
N Accumulation: 200
Normfactor:  1.0076158e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ni
 
exp_x 92.0204010009766 92.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  28   1      7.472   6697597.      2589.     157.32
