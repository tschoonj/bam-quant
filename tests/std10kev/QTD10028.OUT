     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:27:03  บ   
     บ  Spectrum: QTD10028.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        4 iterations done    บ   
     บ  ChiSquare =   390.8   last change =   .09%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -63.2 ๑   100.0   บ     -63.2 ๑      .7   บ   
     บ  GAIN (eV/ch)  บ     20.320 ๑   2.000   บ    20.320 ๑    .002   บ   
     บ det NOISE (eV) บ     -128.6 ๑    40.0   บ    -128.6 ๑    -2.5   บ   
     บ  FANO factor   บ       .262 ๑    .050   บ      .262 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ni-Ka                           6697597. ๑   2589.           บ   
     บ    KA1         7.478    .66126     4428857. ๑   1713.    173.2  บ   
     บ              371.130    158.84        3480.           9.47E-01  บ   
     บ    KA2         7.461    .33874     2268740. ๑    879.    157.3  บ   
     บ              370.293    158.54        3533.           9.47E-01  บ   
     บ                                                                 บ   
     บ  2 Ni-Kb                            969818. ๑    986.           บ   
     บ    KB1         8.265   1.00000      969818. ๑    987.    120.3  บ   
     บ              409.861    172.12        1941.           9.60E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
